whois-api/src/main/java/net/ripe/db/whois/api/elasticsearch/ElasticFullTextIndex.java

```diff
@@ -67,6 +68,38 @@ public void scheduledUpdate() {
         LOGGER.info("Completed updating Elasticsearch indexes");
     }
 
+    @Scheduled(fixedDelay = 60 * 60 * 1_000L)
+    @SchedulerLock(name = ALERT_TASK_NAME)
+    public void checkIndex() {
+        if (!elasticIndexService.isEnabled()) {
+            LOGGER.error("Elasticsearch is not enabled");
+            return;
+        }
+
+        try {
+            alert();
+        } catch (DataAccessException | IOException | IllegalStateException e) {
+            LOGGER.error("Unable to perform fulltext index checks due to {}: {}", e.getClass(), e.getMessage());
+        }
+
+        LOGGER.info("Completed Elasticsearch index check");
+    }
+
+    protected void alert() throws IOException{
+        if (shouldRebuild()) {
```

Comment:
Why you need this condition? If ES is enabled just directly check the objects.

Reply:
This will be true when there is not document in Elasticsearch, or no metadata or not serial.
In those cases I guess that making this check makes no sense